include.js

```diff
@@ -1,4 +1,11 @@
 let cards = [
+  {
+    artName: 'Loading Page',
+    pageLink: './Art/SaumyaKumar-09/Animation%204/index.html',
+    imageLink: './Art/SaumyaKumar-09/Animation%204/Gif4.mp4',
```

Comment:
Please replace the image link with a link to a `.gif` or `.png` file.

Reply:
Ok. Thank you!